tweets/2019/03/ranger-0.tweet

```diff
@@ -0,0 +1,3 @@
+Ranger, a Probot app for open-source maintainers, just landed in the @github marketplace! Ranger helps automate common maintenance tasks, like merging PRs, closing issues, creating tags, deleting comments and branches, and more!
+
+Visit reporanger.com to find out more.
```

Comment:
just to be 100% sure that Twitter will turn this into a link when we post the tweet via its API, maybe add the https?

```suggestion
https://reporanger.com/
```

Reply:
I can do that! I did paste the text into twitter and it turned it into a link. Is that enough to confirm? 

Otherwise I can just make this update too (:

Reply:
Yeah better safe than sorry 🤷‍♂️